supplementary_style_guide/style_guidelines/support.adoc

```diff
@@ -0,0 +1,32 @@
+[[support]]
+= Support
+
+[[technology-preview-guidance]]
+== Technology Preview
+
+Technology Preview features provide early access to upcoming product innovations, enabling customers to test functionality and provide feedback during the development process. However, these features are not fully supported. Documentation for a Technology Preview feature might be incomplete or include only basic installation and configuration information.
+
+When documenting a Technology Preview feature, follow these guidelines:
+
+* Add an admonition labeled IMPORTANT at the beginning of the Technology Preview content and include the template text.
+* Use initial uppercase capitalization, that is, Technology Preview.
+* Include a brief description of the Technology Preview feature in the release notes.
+* Maintain a list of features that are currently in Technology Preview status in the release notes.
+* Never use the phrase "supported as a Technology Preview", and avoid using "support" in Technology Preview descriptions. Instead, use neutral words like "available", "provide", "capability", and so on.
+* When the Technology Preview feature becomes generally available, remove the IMPORTANT admonition from the release notes and any other document that includes content about the feature.
+
+For more information about the support scope of Red Hat Technology Preview
+features, see link:https://access.redhat.com/support/offerings/techpreview/[Technology Preview Features Support Scope].
+
+
+Use the following template. Replace _<feature_name>_ with the feature name:
```

Comment:
@ndeevy @bergerhoffer @leswilliams44 - Sorry to be "that person" that keeps bringing up issues, but in looking over this over again, I was just wondering why we are including a second copy of the TP template here when we already have one in the "Release notes" section. Why not just xref the actual code template we want people to use in the RN section? 

And if there's a case to include the template here, shouldn't it be identical to the one in the "Release notes" section and formatted as code? The currently published template refers to `tech_preview_name` rather than `feature_name` as the user-defined variable.

Reply:
@bburt-rh  It's good to be "that person" if it's valid :-)  and I think it is here. I agree it'd be better not to duplicate it.
To be honest I think the currently published entries for tech preview and deprecated+removed functionality in the release notes section don't really belong there. That subsection was originally for writing doc texts, which is not where the contents of that template would appear anyway right. Maybe that all (tech preview and deprecated) belongs in this new Support section?
I just looked at a few BZs from release notes that have doc texts, and they are just the usual, tech writer-edited descriptions of the feature plus '<x> is available as a Technology Preview'.

Reply:
@ndeevy- Thanks!  @bergerhoffer is submitting a separate PR to address some of these concerns, so we can continue our discussion re that new PR.